tdesign/src/Icon/IconBlankCircle.tsx

```diff
@@ -0,0 +1,6 @@
+import { blankCircleURI } from "./cssSvgStrings";
+import BaseIcon, { IIconProps } from "./BaseIcon";
+
+export const IconBlankCircle = (props: IIconProps) => {
+  return <BaseIcon iconSlug={"all"} svgURI={blankCircleURI} {...props} />;
```

Comment:
The `iconSlug` should probably be something else, e.g.

```suggestion
  return <BaseIcon iconSlug={"blankCircle"} svgURI={blankCircleURI} {...props} />;
```